### PR TITLE
docs(build): update and sync target descriptions

### LINKED ISSUE #1187: Add description to ant targets
The ant target offer documentation via `@description`. We should properly document the targets there

### PR DESCRIPTION
**Pull request by musicEnfanthen**
This PR adds some missing descriptions for the build.xml targets, updates the exisiting ones and syncs them with the descriptions in BUILD_COMMANDLINE.md.

Fixes #1187

### CODE REVIEW

File: BUILD_COMMANDLINE.md [hunk 1]
```diff
@@ -143,14 +143,14 @@ The following targets can be called using `ant <target>`:
 | target                | description     |
 |-----------------------|-----------------|
 | `dist` (or no target) | Default main target; equivalent to calling ant without any target. Builds all artifacts, i.e., RNG and compiled ODDs of all customizations, guidelines html and PDF.  |
-| `canonicalize-source` | Creates a canonicalized version of the mei-source.xml. This target will be triggered before all `build-...` targets. |
-| `build-compiled-odds` | Builds the compiled ODD files for all MEI customizations: `mei-all`, `mei-all_anyStart`, `mei-basic`, `mei-CMN`, `mei-Mensural` and `mei-Neumes`. |
-| `build-compiled-odd -Dcustomization.path="[ABSOLUTE/PATH/TO/YOUR/CUSTOMIZATION]"` | Builds the compiled ODD of a specific customization. |
+| `canonicalize-source` | Creates a canonicalized version of the mei-source.xml, i.e. resolves xincludes and puts result in `build/mei-source_canonicalized`. This target will be triggered before all `build-...` targets. |
```
Comment (rettinghaus) [inline BUILD_COMMANDLINE.md]:
```suggestion
| `canonicalize-source` | Creates a canonicalized version of the mei-source.xml, i.e., resolves xincludes and puts result in `build/mei-source_canonicalized`. This target will be triggered before all `build-...` targets. |
```

File: build.xml [hunk 2]
```diff
@@ -326,29 +326,29 @@
...
 
-    <target name="generate-images" description="Generate SVG images of mei files in assets/images/GeneratedImages; depends on docker-mei container image" if="${docker}">
+    <target name="generate-images" description="Generates SVG images of mei files in assets/images/GeneratedImages; depends on docker-mei container image." if="${docker}">
         <exec executable="node" dir="${basedir}/../">
             <arg value="index.js"/>
         </exec>
     </target>
     
-    <target name="generate-images-py">
+    <target name="generate-images-py" description="Generates SVG images of mei files in assets/images/GeneratedImages; depends on python3 with verovio installed.">
```
Comment (rettinghaus) [inline build.xml]:
```suggestion
    <target name="generate-images-py" description="Generates SVG images of MEI files in assets/images/GeneratedImages; depends on python3 with Verovio installed.">
```

**Review by rettinghaus — CHANGES_REQUESTED** [review]
Looks almost perfect. 🤓

File: build.xml [hunk 3]
```diff
@@ -199,7 +199,7 @@
         <get src="https://www.oxygenxml.com/maven/com/oxygenxml/oxygen-basic-utilities/${xerces.version}/${oxygen.basic.utilites.jar.file}" dest="${dir.lib.xerces}"/>
     </target>
 
-    <target name="canonicalize-source" description="Canonicalizes the mei-source.xml, i.e. resolves xincludes and puts result in build/mei-source_canonicalized.xml" depends="init-mei-classpath" >
+    <target name="canonicalize-source" description="Creates a canonicalized version of the mei-source.xml, i.e. resolves xincludes and puts result in build/mei-source_canonicalized. This target will be triggered before all `build-...` targets." depends="init-mei-classpath" >
```
Comment (musicEnfanthen) [inline build.xml]:
```suggestion
    <target name="canonicalize-source" description="Creates a canonicalized version of the mei-source.xml, i.e., resolves xincludes and puts result in build/mei-source_canonicalized. This target will be triggered before all `build-...` targets." depends="init-mei-classpath" >
```

File: build.xml [hunk 4]
```diff
@@ -326,29 +326,29 @@
         </copy>
     </target>
 
-    <target name="generate-images" description="Generate SVG images of mei files in assets/images/GeneratedImages; depends on docker-mei container image" if="${docker}">
+    <target name="generate-images" description="Generates SVG images of mei files in assets/images/GeneratedImages; depends on docker-mei container image." if="${docker}">
```
Comment (musicEnfanthen) [inline build.xml]:
```suggestion
    <target name="generate-images" description="Generates SVG images of MEI files in assets/images/GeneratedImages; depends on docker-mei container image." if="${docker}">
```

File: BUILD_COMMANDLINE.md (hunk 1 above)
Comment (musicEnfanthen) [reply BUILD_COMMANDLINE.md]: Thanks. Updated in build.xml as well.

File: build.xml (hunk 2 above)
Comment (musicEnfanthen) [reply build.xml]: Thanks, updated in parallel description as well.

**Review by bwbohl — APPROVED** [review]

**Review by rettinghaus — APPROVED** [review]

**Pull request merged**